lib/datadog/appsec/contrib/devise/event_information.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Devise
+        # Module to extract event information from the resource
+        module EventInformation
```

Comment:
WDYT of the class name simply be `Devise::Event`?

Reply:
Sounds good